person.rb

```diff
@@ -1,18 +1,31 @@
-class Person
-  attr_accessor :name, :age
-  attr_reader :id
+require_relative('./corrector')
 
+class Person
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @corrector = Corrector.new
+    @rentals = []
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+  attr_accessor :rentals
```

Comment:
- [ ] [OPTIONAL] you can add the  `attr_accesor` for `:rentals`  in the same line as `:name` and `:age`